lua/toggleterm.lua

```diff
@@ -184,6 +184,80 @@ function M.exec(cmd, num, size, dir, direction, go_back, open)
   term:send(cmd, go_back)
 end
 
+--- @param selection_type string
+--- @param trim_spaces boolean
+--- @param terminal_id number
+function M.send_lines_to_terminal(selection_type, trim_spaces, terminal_id)
+  -- trim_spaces defines if we should trim the spaces from lines which are sent to the terminal
+  if trim_spaces == nil then trim_spaces = true end
```

Comment:
nit: `stylua` will reformat this into 3 line statements, whereas you could do 
`trim_spaces = trim_spaces == nil and true or trim_spaces`, is more concise

Reply:
done